src/postmeta/wp-postmeta-detectlinks.js

```diff
@@ -0,0 +1,50 @@
+const { __ } = wp.i18n;
+const { compose } = wp.compose;
+const { withSelect, withDispatch } = wp.data;
+
+const { PluginDocumentSettingPanel } = wp.editPost;
+const { ToggleControl, PanelRow } = wp.components;
```

Comment:
in ES6 import syntax, it uses, for example `import { compose } from '@wordpress/compose'`, same to the others

Reply:
done